package.json

```diff
@@ -44,7 +44,7 @@
     "dev:invoke": "(cd packages/airnode-node && yarn run dev:invoke)",
     "dev:list": "(cd packages/airnode-operation && yarn run dev:list)",
     "dev:stop": "(cd packages/airnode-operation && yarn run dev:stop)",
-    "docker": "yarn docker:build:artifacts && yarn docker:build:deployer && yarn docker:build:client && yarn docker:build:admin",
+    "docker:build": "yarn docker:build:artifacts && yarn docker:build:deployer && yarn docker:build:client && yarn docker:build:admin",
```

Comment:
Not sure why we call it just `docker` and all the "subtasks" are called `docker:build:*` so I changed it.